### PR TITLE
More Symfony 5 compatibility fixes

### PR DESCRIPTION
**Pull request by aboks**
| Q             | A
| ------------- | ---
| Bug fix?      | Yes
| New feature?  |  No
| BC breaks?    | No
| Deprecations? | No
| Fixed tickets | N/A

I hate to bring the bad news here after your previous efforts to support Symfony 5, but there are still incompatibilities with symfony/process` 5.x in the Rsync- and SSH-client. This PR should fix them.

> Do not forget to add notes about your changes to CHANGELOG.md
>
> Easiest way to do it, by running next command:
>
>     php bin/changelog
>

### CODE REVIEW

**Comment by loevgaard** [comment]
Could this be merged and released? :)

File: src/Utility/Rsync.php [hunk 1]
```diff
@@ -43,7 +43,11 @@ public function call($hostname, $source, $destination, array $config = [])
 
         $this->pop->command($hostname, $rsync);
 
-        $process = new Process($rsync);
+        if (method_exists('Symfony\Component\Process\Process', 'fromShellCommandline')) {
```
Comment (sirian) [inline src/Utility/Rsync.php]: Maybe replace with `if (method_exists(Process::class, 'fromShellCommandline'))`?

File: src/Ssh/Client.php [hunk 2]
```diff
@@ -174,4 +174,13 @@ private function exec($command, &$exitCode = null)
         }
         return $output;
     }
+
+    private function createProcess($command)
+    {
+        if (method_exists('Symfony\Component\Process\Process', 'fromShellCommandline')) {
```
Comment (sirian) [inline src/Ssh/Client.php]: Maybe replace with `if (method_exists(Process::class, 'fromShellCommandline'))`?

File: src/Utility/Rsync.php (hunk 1 above)
Comment (aboks) [reply src/Utility/Rsync.php]: I agree that would be slightly cleaner. For now I used the exact same fix that was already present in e.g. `Deployer\Utility\ProcessRunner` for consistency. The best long-term solution would probably be to have a utility function somewhere in the deployer codebase to contain this compatibility layer.

**Pull request merged**